src/middleware/signed-trx-tx-middleware.ts

```diff
@@ -0,0 +1,50 @@
+import debug from 'debug'
+import { TronWebSigner } from '../tron'
+
+import { SignedTx } from '../proto/loom_pb'
+import { ITxMiddlewareHandler } from '../client'
+import { EthersSigner, soliditySha3 } from '../solidity-helpers'
+import { bytesToHex, hexToBytes } from '../crypto-utils'
+import { Address } from '../address'
+
+const log = debug('signed-trx-tx-middleware')
+
+/**
+ * Signs transactions using an Ethereum compatible (secp256k1) private key.
+ */
+export class SignedTrxTxMiddleware implements ITxMiddlewareHandler {
```

Comment:
Probably better to call this `SignedTronTxMiddleware`, `TrxTx` is a bit awkward to type